domains/aman-sapkota.json

```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "dev-amansapkota",
+    "email": "amansapkota14@gmail.com"
+  },
+  "records": [
+    {
+      "type": "CNAME",
+      "value": "aman-sapkota.github.io"
+    }
+  ]
```

Comment:
Improper way to do a CNAME record. https://docs.is-a.dev/ Reopen PR when this is fixed.